bcipy/language/model/gpt2.py

```diff
@@ -23,45 +24,143 @@ def __init__(self, response_type, symbol_set, lm_path=None):
         super().__init__(response_type=response_type, symbol_set=symbol_set)
         self.model = None
         self.tokenizer = None
-        self.is_start_of_word = True
         self.vocab_size = 0
         self.idx_to_word = None
-        self.curr_word_predicted_prob = None
         self.lm_path = lm_path or "gpt2"
+
+        # Hard coding a unigram language model trained on ALS phrase dataset
+        # for smoothing purpose
+        self.unigram_lm = {'E': 0.0998, 'T': 0.096, 'O': 0.0946, 'I': 0.0835,
```

Comment:
Would this fix our symbol set? This may be alright given how we want to use it in the short term, but we should validate the same set was passed as you define here.

Reply:
This serves as a quick way to introduce a unigram language model for interpolating purpose. It does use the same symbol set as the default symbol set, alphabet(). I think for now this should be alright. If the symbol set is a parameter that would change, we probably need to train a unigram model on that symbol set on the fly, which may be undesirable. For now I have added a check to make sure that the unigram symbol set is the same as the one passed in.